scripts/script-F-postprocessing-upload.pl

```diff
@@ -55,24 +55,19 @@
 	print " OK\n";
 	my $destfile = $props->{'Publishing.UploadTarget'} . '/' . $props->{'Fahrplan.ID'} . "-" . 
 		$props->{'EncodingProfile.Slug'} . '.' . $props->{'EncodingProfile.Extension'};
-	# support old property as fallback
-	my $opts = $props->{'Publishing.UploadOptions'};
-	$opts = $props->{'Processing.Postprocessing.Options'} unless defined ($opts);
-	$opts = "" unless defined ($opts);
 	print "$srcfile (in) -> $destfile (out) ...";
-	my $cmd = "scp -B -p $opts '$srcfile' '$destfile'";
+	# rsync: verbose, '-e ssh', keep partially transferred files, keep mtimes
+	my $cmd = "rsync --verbose --rsh=ssh --partial --times '$srcfile' '$destfile'";
```

Comment:
shouldn't there be at least an "-a" in some form, or what am I missing?

Reply:
Well, we don't need `-r`ecursive, `-l`inks, modification `-t`imes, `-D`evices and specials, and can't set  `-g`roups, `-o`wner and `-p`erms anyway, so setting `-a` is not needed IMHO.